gm_dotnet_managed/GmodNET/Startup.cs

```diff
@@ -166,6 +166,18 @@ internal static class Startup
 
                 ILua lua = new Lua(lua_base);
 
+                lua.PushSpecial(SPECIAL_TABLES.SPECIAL_GLOB);
+                lua.GetField(-1, "print");
+                lua.PushString(
+                    "   _____       _   _   ______   _______ \n"+
+                    "  / ____|     | \\ | | |  ____| |__   __|\n"+
+                    " | |  __      |  \\| | | |__       | |   \n"+
+                    " | | |_ |     | . ` | |  __|      | |   \n"+
+                    " | |__| |  _  | |\\  | | |____     | |   \n"+
+                    "  \\_____| (_) |_| \\_| |______|    |_|   \n");
+                lua.MCall(1,0);
+                lua.Pop();
```

Comment:
.Pop(1) please if possible to follow the rest of the calling conventions

Reply:
resolved in https://github.com/GmodNET/GmodDotNet/pull/74/commits/e3471759fc7e8196e0b0fd4be3fd1add3a6b807b